changelog.txt

```diff
@@ -35,6 +35,7 @@
 * Fix - Misleading subscription order note on payment method change.
 * Fix - Incorrect error message when card ZIP validation fails.
 * Add - `Requires PHP` and `Requires at least` to the main plugin file.
+* Fix - Prevent quick-pay buttons from showing up in Composite Product pages. 
```

Comment:
Might change "quick-pay" to "Payment Request" to make it consistent with other log entries? 

Also, can you add this changelog entry for readme.txt, and move it to version 3.4.0?

Reply:
Thank you @htdat for your feedback!